com.ibm.wala.cast.js/src/testFixtures/java/com/ibm/wala/cast/js/test/ExtractingToPredictableFileNames.java

```diff
@@ -0,0 +1,26 @@
+package com.ibm.wala.cast.js.test;
+
+import com.ibm.wala.cast.js.html.DomLessSourceExtractor;
+import com.ibm.wala.cast.js.html.JSSourceExtractor;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ExtractingToPredictableFileNames implements AutoCloseable {
```

Comment:
I think I get it, but can we add some Javadoc indicating the purpose of this class?

Reply:
We can, we should, and I will. Thanks for not letting me be lazy about documentation, @msridhar.